App.js

```diff
@@ -2,17 +2,84 @@ import 'react-native-gesture-handler';
 import { createStackNavigator } from '@react-navigation/stack';
 import { NavigationContainer} from '@react-navigation/native';
 import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Dashboard from './screens/Dashboard';
 
-import Dashboard from './screens/Dashboard'
 
 const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
+
+function DashboardStack() {
+  return (
+    <Stack.Navigator
+      initialRouteName="Dashboard"
+      screenOptions={{
+        headerStyle: { backgroundColor: '#3DD5F4' },
+        headerTintColor: '#fff',
+        headerTitleStyle: { fontWeight: 'bold' },
+      }}>
+      <Stack.Screen
+        name="Dashboard"
+        component={Dashboard}
+        options={{ title: 'CoSava Dashboard' }}
+      />
+    </Stack.Navigator>
+  );
+}
+
+/*function SettingsStack() {
```

Comment:
Is this function intended to be used in the future? If not so, we should get rid of it.

Reply:
if we have a settings page then it will be used, it won't be used if we don't have a settings page. I can delete for now